### PR TITLE
[v2][www] Add global layout to pages and templates that don’t have it yet.

### PR DESCRIPTION
**Pull request by ryanditjia**
Plus fixes to a couple of small issues.

### CODE REVIEW

File: www/src/pages/features.js [hunk 1]
```diff
@@ -95,7 +95,6 @@ const LegendTable = () => {
           border: `1px solid ${legendBorderColor}`,
           borderLeft: 0,
           fontFamily: options.headerFontFamily.join(`,`),
-          display: `table`,
```
Comment (ryanditjia) [inline www/src/pages/features.js]: This is a duplicate of line 94.

File: www/src/templates/tags.js [hunk 2]
```diff
@@ -46,7 +46,6 @@ Tags.propTypes = {
         PropTypes.shape({
           node: PropTypes.shape({
             frontmatter: PropTypes.shape({
-              path: PropTypes.string.isRequired,
```
Comment (ryanditjia) [inline www/src/templates/tags.js]: This prop is neither queried nor used by the component.

**Comment by m-allanson** [comment]
Great stuff! Deploy preview can be seen here: https://deploy-preview-5454--gatsbyjs.netlify.com/

**Comment by m-allanson** [comment]
There's a failing test for this PR, although it doesn't seem related. Can you replicate that failure locally?

**Comment by m-allanson** [comment]
The tests seem to work with an updated lock file. Try removing `yarn.lock` and `node_modules`, then running `yarn` and `yarn run bootstrap` to fix?

**Pull request merged**